src/app/components/team/Statuses/EditStatusDialog.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { mountWithIntl } from '../../../../../test/unit/helpers/intl-test';
+import EditStatusDialog from './EditStatusDialog';
+
+describe('<EditStatusDialog />', () => {
+  const team = {
+    slug: 'new-team',
+    id: '1',
+    team_bot_installations: {
+      edges: [],
+    },
+  };
+
+  const team2 = {
+    slug: 'new-team',
+    id: '1',
+    team_bot_installations: {
+      edges: [{
+        node: { team_bot: { identifier: 'smooch' } },
+      }],
+    },
+  };
+
+  const status = {
+    id: '1',
+    locales: {
+      en: { label: 'True', description: 'description A', message: 'message to be sent' },
+    },
+    style: { color: 'black' },
+    label: 'True',
+    should_send_message: true,
+  };
+
+  it('should render "Add a new status" title when do not pass status', () => {
+    const wrapper = mountWithIntl(<EditStatusDialog
+      open
+      team={team}
+      defaultLanguage="English"
+      onCancel={() => {}}
+      onSubmit={() => {}}
+      handleSelect={() => {}}
+    />);
+    expect(wrapper.html()).toMatch('Add a new status');
+    expect(wrapper.html()).not.toMatch('Edit status');
+  });
+
+
+  it('should render "Edit status" title when pass status', () => {
+    const wrapper = mountWithIntl(<EditStatusDialog
+      status={status}
+      team={team}
+      defaultLanguage="en"
+      defaultValue={status}
+      selectedStatus={status}
+      open={Boolean(status)}
+      onCancel={() => {}}
+      onSubmit={() => {}}
+      handleSelect={() => {}}
+    />);
+    expect(wrapper.html()).not.toMatch('Add a new status');
+    expect(wrapper.html()).not.toMatch('Send a message to the user who requested the item when you change an item to this status');
+    expect(wrapper.html()).toMatch('Edit status');
+    expect(wrapper.html()).toMatch('True');
+    expect(wrapper.html()).toMatch('description A');
+    expect(wrapper.find('#edit-status-dialog__status-message-label').hostNodes()).toHaveLength(0);
+  });
+
+
+  it('should render message field when smooch is installed on the team', () => {
+    const wrapper = mountWithIntl(<EditStatusDialog
+      status={status}
+      team={team2}
+      defaultLanguage="en"
+      defaultValue={status}
+      selectedStatus={status}
+      open={Boolean(status)}
+      onCancel={() => {}}
+      onSubmit={() => {}}
+      handleSelect={() => {}}
+      margin="normal"
+      fullWidth
+      maxWidth="xs"
```

Comment:
`margin`, `fullWidth` and `MaxWidth` aren't props of `EditStatusDialog`, please remove these.